openslides_backend/action/actions/meeting_user/helper.py

```diff
@@ -17,4 +18,8 @@ def create_or_get_meeting_user(self, meeting_id: int, user_id: int) -> int:
                 MeetingUserCreate,
                 [{"meeting_id": meeting_id, "user_id": user_id}],
             )
-            return action_results[0]["id"]  # type: ignore
+            id_ = action_results[0]["id"]  # type: ignore
+            self.datastore.changed_models.get(
+                fqid_from_collection_and_id("meeting_user", id_), {}
+            ).pop("meta_new", None)
```

Comment:
Why do you remove the `meta_new` here?

Reply:
Otherwise I think, will be created again